_posts/2021-03-05-scala-3-developers-preview-survey-results.md

```diff
@@ -0,0 +1,328 @@
+
+Since announcing this news (ScalaDays Berlin 2018), it has taken a good 2+ years to finalize the changes,
+sync the community’s efforts, publish migration paths, and create a new ecosystem of tools and libraries.
+We are very near to being able to take the final step - releasing Scala 3.0.
+We are still not there yet but are closer than ever.
+
+In December 2020 Scala 3.0.0-M3 was released, the last milestone before the RC1 release, scheduled for January 2021.
+
+Scala 3 release teams were ready to move forward but still needed
+a final feedback loop from the community to do so. We published a blog on December 15th 2020,
+sharing the status of the Scala 3 release, and to ask the Scala community 
+to give us their opinions, concerns and advice.
+
+Today, we are pleased to share what we learned with you.
+First, we want to thank everyone who participated and selflessly poured
+in such a rich amount of insights and knowledge our way! Based on this survey,
+we will take actions and lead conversations. So, keep following and sharing,
+the quality of Scala development depends on your feedback.
+
+Below, we will present our findings in 3 sections:
+1. _**About the survey**_ - contains all information about 
+   creating the survey and collecting data, and methodology for our analysis.
+2. _**Presenting the data**_ - summarizes the data we received, mostly quantifiable
+3. _**Conclusions**_ - this is the part that shows 
+   how we read the data and what steps we will take based on our reading.
+
+
+Conclusions are open to interpretation, which is why we offer full
+insight into our methodology and data, so feel free to make your own and share it with us.
+
+## 1 - About the Survey
+The survey has a simple design:
+- A total of 4 questions and a free field at the end
+- Each question contained a “closed” and an “open” part
+  Please check what it looks like [here](https://docs.google.com/forms/d/e/1FAIpQLSflVmTu9lhrtnSTh2tKAjUGrt3WvEgwlDqZg66O3EVSXd1aJg/viewform).
+
+The survey questions focused on Scala 3 features, tooling, migration, and communication.
+We chose this format for several reasons:
+- Simple and short - we wanted the participants to have a “quick but detailed” experience, 
+  in order to get as many answers as possible, and be as detailed as possible
+- Allow our teams to quantify and qualify the answers
+- Allow our teams to identify areas of improvement
+- Can point to important conversations our teams should open/pay attention to
+- Within the framework we provided, we can easily cross-reference/compare 
+  results with other platforms where community shares their views
+
+Our aim was to maximise our reach towards potential participants 
+within the Scala community. To achieve this, the main channels of distribution were: 
+the above-mentioned blog article, Twitter (scala_lang), the Contributors forum, various Slack channels, and mailing lists.
+
+Since 14th January 2021 and even until publishing these results, answers are 
```

Comment:
I think "Since 14th January 2021 and even until publishing these results" could just be omitted. Maybe add "so far" after "the answer activity" in the next sentence.